core/domain/story_domain.py

```diff
@@ -0,0 +1,359 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Domain objects relating to stories."""
+
+from constants import constants
+from core.platform import models
+import feconf
+
+(story_models,) = models.Registry.import_models([models.NAMES.story])
+
+
+# Do not modify the values of these constants. This is to preserve backwards
+# compatibility with previous change dicts.
+STORY_PROPERTY_TITLE = 'title'
+STORY_PROPERTY_DESCRIPTION = 'description'
+STORY_PROPERTY_TOPIC = 'topic'
+STORY_PROPERTY_NOTES = 'notes'
+STORY_PROPERTY_LANGUAGE_CODE = 'language_code'
+STORY_PROPERTY_STORY_CONTENTS = 'story_nodes'
```

Comment:
Match the constant name to the string. Don't call one "story contents" and the other "story nodes".

Reply:
Done, renamed both to story_nodes.